tests/test_generate_schema.py

```diff
@@ -0,0 +1,351 @@
+import re
+from math import isnan
+from glob import glob
+import os
+import shutil
+
+import pytest
+from dmutils.content_loader import ContentQuestion
+from hypothesis.settings import Settings
+from hypothesis import settings, given, assume, strategies as st
+from scripts.generate_schemas import uri_property, parse_question_limits, \
+    checkbox_property, percentage_property, multiquestion, \
+    build_question_properties, empty_schema, load_questions, \
+    drop_non_schema_questions, radios_property, list_property, price_string, \
+    pricing_property, generate_schema, SCHEMAS
+from scripts import support
+
+
+# Python 3 compatibility
+try:
+    xrange
```

Comment:
Is there a problem with just using python2 `range`? It looks like we're building a list from it either way